lib/galaxy/config.py

```diff
@@ -113,6 +113,7 @@ def __init__( self, **kwargs ):
 
         self.expose_user_name = kwargs.get( "expose_user_name", False )
         self.expose_user_email = kwargs.get( "expose_user_email", False )
+        self.password_expiration_period = timedelta(days=int(kwargs.get( "password_expiration_period", 0 )))
```

Comment:
please adhere to the codestyle of the file and add space before and after a bracket content